simpleeval.py

```diff
@@ -112,8 +112,22 @@
 # people not be stupid.  Allowing these functions opens up all sorts of holes - if any of
 # their functionality is required, then please wrap them up in a safe container.  And think
 # very hard about it first.  And don't say I didn't warn you.
+# builtins is a dict in python >3.6 but a module before
+if isinstance(__builtins__, dict) and 'help' in __builtins__: 
+    DISALLOW_FUNCTIONS = {
+        type, isinstance, eval, getattr, setattr, help, repr, compile, open
+        }
+elif 'help' in dir(__builtins__):
+    DISALLOW_FUNCTIONS = {
+        type, isinstance, eval, getattr, setattr, help, repr, compile, open
+        }
+else:
+    print('help not in builtins')
+
+    DISALLOW_FUNCTIONS = {
+        type, isinstance, eval, getattr, setattr, repr, compile, open
+        }
```

Comment:
Could you please change this to something like:
```suggestion
DISALLOW_FUNCTIONS = {
        type, isinstance, eval, getattr, setattr, repr, compile, open
        }
if hasattr(__builtins__, 'help') or 'help' in __builtins__:
    # PyInstaller environment doesn't include this module.
    DISALLOW_FUNCTIONS.add(help)
```

Type of thing?

Great catch finding this edge case 👍